test/client/client_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package client_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestClient(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Client Suite")
```

Comment:
IIRC correct we configure the junit reporter for E2E tests. I suggest to consider the similar ginkgo register routines in our E2E test suite.

Reply:
Yep. The J2EE reporter and the collection of debugging artifacts should also move to Go in a later PR.